api/app/main.py

```diff
@@ -86,8 +88,33 @@
 @app.get('/health')
 async def get_health():
     """ A simple endpoint for Openshift Healthchecks """
-    LOGGER.info('/health')
-    return {"message": "Healthy as ever"}
+    try:
+        parts = [
+            'https://console.pathfinder.gov.bc.ca:8443/apis/apps/v1beta1/namespaces/',
```

Comment:
I sneaked in an environment variable, could you please us that?: PATHFINDER_BASE_URI